### PR TITLE
feat: add basic home page

### PR DESCRIPTION
**Pull request by NTag**
This PR adds:
- a simple homepage, allowing to search;
- an API route `/api/:q` returning an array of results. This is currently a fake and stupid one. It'll use the lib later.

Once the lib has been published, the types will be imported from it instead of redefining them.

CleanShot 2022-01-15 at 16 07 04@2x
CleanShot 2022-01-15 at 16 07 15@2x

### CODE REVIEW

File: components/Home/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  }, [query]);
+
+  return (
+    <div>
+      <Head>
+        <title>howmuch.green</title>
+        <meta name="description" content="How much green?" />
+      </Head>
+      <S.Container>
+        <S.Title>🌱 how much green is:</S.Title>
```
Comment (licarth) [inline components/Home/index.tsx]: I think you can't say "How much green is ...?", but rather "How green is …?"

Comment (NTag) [reply components/Home/index.tsx]: I wanted to stick to the domain name, but maybe it's better to write a correct sentence

File: components/Home/index.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        <form onSubmit={onSubmit}>
+          <S.Input
+            placeholder="iPho…"
+            value={query}
+            onChange={(e) => setQuery(e.target.value)}
+          />
+        </form>
+        <S.ResultsContainer>
+          {results.map((result, i) => (
+            <Link href={`/${result.name.replace(" ", "")}`} key={i} passHref>
```
Comment (licarth) [inline components/Home/index.tsx]: This will replace only the first space in `result.name`

**Review by licarth — APPROVED** [review]

File: components/Home/index.tsx (hunk 2 above)
Comment (NTag) [reply components/Home/index.tsx]: Rah it's true I hate this

**Pull request merged**